### PR TITLE
REF: BlueListemItem to TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/j665xdtlgoktuzucapt64qzosu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12222)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/u5sbcttdwra6rkynu62xhm75uu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12223)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/lmdkeumybalgbl4k23mtoiqtnm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12228)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: screen/send/coinControl.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useMemo, useState, useContext, useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
-import { Avatar, Badge, Icon, ListItem } from 'react-native-elements';
+import { Avatar, Badge, Icon, ListItem as RNElementsListItem } from 'react-native-elements';
```
Comment (Overtorment) [inline screen/send/coinControl.js:3]: if you keep this ListItem as is, without renaming, and rename  our own ListItem to BlueListItem, the total diff will be smaller

File: screen/send/details.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ import ToolTipMenu from '../../components/TooltipMenu';
 import { requestCameraAuthorization, scanQrHelper } from '../../helpers/scan-qr';
 import { useTheme } from '../../components/themes';
 import Button from '../../components/Button';
+import ListItem from '../../components/ListItem';
```
Comment (Overtorment) [inline screen/send/details.js:46]: up to you, but here as well you can import it as `BlueListItem` and you wont have to change the whole file

**Review by Overtorment — CHANGES_REQUESTED** [review]
conflict

File: screen/send/coinControl.js (hunk 1 above)
Comment (marcosrdz) [reply screen/send/coinControl.js:3]: i was under the impression that we wanted to remove the Blue prefix from components

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rfwo4ly54yulmyacgcsynrhbqq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12233)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ww7nzsep4ppmtsxshjrg6foide?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12234)

**Pull request merged**